code/mob/living/critter/ai.dm

```diff
@@ -197,6 +193,14 @@ var/list/ai_move_scheduled = list()
 	proc/was_harmed(obj/item/W, mob/M)
 		.=0
 
+	proc/disable()
+		src.enabled = FALSE
+		src.stop_move()
+
+	proc/enable()
+		src.enabled = TRUE
+		src.interrupt()
+
```

Comment:
maybe also set `is_npc` here? although that would probably be better off as a proc/macro anyway

Reply:
I'm not sure the ai should be touching `is_npc`. Its use is a bit ambiguous, I was gonna go through it and clean up the references to it in another PR. I'm thinking `is_npc` should literally just be a flag for "this is an active mob that isn't a player" and have `ai.enabled` indicate whether the AI should tick.